infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -396,6 +394,15 @@ private boolean runRecoveryReconfiguration() {
         getCorfuRuntime().invalidateLayout();
         Layout clusterLayout = getCorfuRuntime().getLayoutView().getLayout();
 
+        // Trigger sequencer recovery only if this node was successful in committing the recovery
+        // layout. If not, then we can assume that the sequencer is already bootstrapped.
+        // NOTE: This is an async task and the failure of this task does not affect the recovery.
+        if (recoveryReconfigurationResult) {
```

Comment:
Why does moving the triggerSequenceBootstrap here make a difference? previously in L290 we would skip triggering the boot strap if recovery result was unsuccessful.